components/list/list-item.js

```diff
@@ -1,101 +1,18 @@
 import '../colors/colors.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
-import { checkboxStyles } from '../inputs/input-checkbox-styles.js';
-import { classMap} from 'lit-html/directives/class-map.js';
-import { getFirstFocusableDescendant } from '../../helpers/focus.js';
-import { getUniqueId } from '../../helpers/uniqueId.js';
-import { ifDefined } from 'lit-html/directives/if-defined.js';
-import ResizeObserver from 'resize-observer-polyfill';
-import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { ListItemMixin } from './list-item-mixin.js';
 
-const ro = new ResizeObserver(entries => {
-	entries.forEach(entry => {
-		if (!entry || !entry.target || !entry.target.resizedCallback) {
-			return;
-		}
-		entry.target.resizedCallback(entry.contentRect && entry.contentRect.width);
-	});
-});
-
-const defaultBreakpoints = [842, 636, 580, 0];
-
-class ListItem extends RtlMixin(LitElement) {
+class ListItem extends ListItemMixin(LitElement) {
```

Comment:
The new list item. I wanted to see how customizations would work. So it defines the image pattern. It add the illustration capabilities to the list item.